src/util/delay.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env jest */
+
+import delay from 'src/util/delay'
+jest.useFakeTimers()
+
+describe('delay tests', () => {
+    test('should return with the given delay', () => {
+        expect.assertions(2)
+        delay(1000)
+        expect(setTimeout.mock.calls.length).toBe(1)
+        expect(setTimeout.mock.calls[0][1]).toBe(1000)
+    })
+})
```

Comment:
It could also be implemented as a separate test. I'm not sure if this test still adds value if a more complete test is added.